2020/7xxx/CVE-2020-7354.json

```diff
@@ -1,18 +1,105 @@
 {
-    "data_type": "CVE",
-    "data_format": "MITRE",
-    "data_version": "4.0",
     "CVE_data_meta": {
+        "ASSIGNER": "cve@rapid7.com",
+        "DATE_PUBLIC": "2020-05-21T13:13:00.000Z",
         "ID": "CVE-2020-7354",
-        "ASSIGNER": "cve@mitre.org",
-        "STATE": "RESERVED"
+        "STATE": "PUBLIC",
+        "TITLE": "Rapid7 Metasploit Pro Stored XSS in 'host' field"
     },
+    "affects": {
+        "vendor": {
+            "vendor_data": [
+                {
+                    "product": {
+                        "product_data": [
+                            {
+                                "product_name": "Metasploit Pro",
+                                "version": {
+                                    "version_data": [
+                                        {
+                                            "version_affected": "<=",
+                                            "version_name": "4.17.1-20200427",
+                                            "version_value": "4.17.1-20200427"
+                                        }
+                                    ]
+                                }
+                            }
+                        ]
+                    },
+                    "vendor_name": "Rapid7"
+                }
+            ]
+        }
+    },
+    "credit": [
+        {
+            "lang": "eng",
+            "value": "Andrea Valenza at the University of Genoa discovered and reported this issue to Rapid7"
+        }
+    ],
+    "data_format": "MITRE",
+    "data_type": "CVE",
+    "data_version": "4.0",
     "description": {
         "description_data": [
             {
                 "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+                "value": "Cross-site Scripting (XSS) vulnerability in the 'host' field of a discovered scan asset in Rapid7 Metasploit Pro allows an attacker with a specially-crafted network service of a scan target store an XSS sequence in the Metasploit Pro console, which will trigger when the operator views the record of that scanned host in the Metasploit Pro interface. This issue affects Rapid7 Metasploit Pro version 4.17.1-20200427 and prior versions, and is fixed in Metasploit Pro version 4.17.1-20200514.\n\nSee also CVE-2020-7354, which describes a similar issue, but involving the generated 'notes' field of a discovered scan asset."
```

Comment:
Shouldn't it be CVE-2020-7355 which affects the 'notes' field?

Reply:
Derp, of course.

Reply:
7354 should be for 'host', 7355 should be 'notes' -- is this not the case? Copy pasting nearly the same thing always screws up something somewhere, but basically:

* CVE-2020-7354 should have the 'host' title and description, but also reference the 'notes' bug in 7355.
* CVE-2020-7355 should have the 'notes' title and description, but also reference the 'host' bug in 7354.

Is this not the case here? I've read it like 10 times and not seeing a mis-identification here (after my latest commit). You /should/ have the rights to edit the msp-xss branch if you can see something I'm not seeing.

Reply:
Re-reviewing, this looks good to me, now. Thanks for the review, @djordan-r7 !